src/go/wsl-helper/pkg/integration/docker_linux.go

```diff
@@ -30,11 +31,14 @@ import (
 
 const (
 	pluginDirsKey = "cliPluginsExtraDirs"
+	credsStoreKey = "credsStore"
+
+	DockerCredentialWinCredExe = "wincred.exe"
```

Comment:
If this is exported it probably should get docs?
(I realize it's only exported for tests…)

Reply:
I'll hard-code it in the test instead. May be better anyways.